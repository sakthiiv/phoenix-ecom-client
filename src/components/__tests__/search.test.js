import React from "react";
import { shallow } from "enzyme";
import Search from "../search";

describe("User", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Search />);
    expect(wrapper).toMatchSnapshot();
  });
});
