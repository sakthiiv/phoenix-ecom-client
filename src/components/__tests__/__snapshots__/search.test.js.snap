// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User renders correctly 1`] = `
<div>
  <h1
    style={
      Object {
        "marginLeft": "20px",
      }
    }
  >
    Search the product
  </h1>
  <form
    className="col-sm-6"
    onChange={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <input
        className="form-control"
        id="search"
        name="search"
        placeholder="Enter name of the product"
        required={true}
        type="text"
      />
    </div>
    <input
      type="submit"
      value="Submit"
    />
  </form>
  <div
    className="col-md-12 plist_container"
    style={
      Object {
        "marginTop": "40px",
      }
    }
  />
</div>
`;
